app/views/works/index.html.erb

```diff
@@ -0,0 +1,33 @@
+<main>
+  <h2>List of Media</h2>
+
+  <% ["album", "book", "movie"].each do |category| %>
+    <section class="media-table">
+      <h4><%= category.capitalize << "s"%></h4>
+      <table class="table">
+        <thead>
+          <tr>
+            <th>Votes</th>
+            <th>Title</th>
+            <th>Created By</th>
+            <th>Published</th>
+            <th>Upvote</th>
+          </tr>
+        </thead>
+        <tbody>
+        <% @works.where(category: category).sort_by_votes.each do |medium| %>
+          <tr>
+            <td><%= medium.votes.count %></td>
+            <td><%= link_to medium.title, work_path(medium) %></td>
+            <td><%= medium.creator %></td>
+            <td><%= medium.publication_year %></td>
+            <td><%= link_to "Upvote",
+                            upvote_path(medium),
+                            method: :post, class: "btn btn-primary"%></td>
+          </tr>
+        <% end %>
```

Comment:
Do you notice that you repeat this type of loop through a set of works over and over again...  Maybe you could put this into a partial and dry up your code.